webapp/browsers.json

```diff
@@ -39,6 +39,14 @@
         "os_name": "linux",
         "os_version": "*"
     },
+    "firefox-58.0-linux": {
+        "initially_loaded": true,
+        "currently_run": true,
```

Comment:
I'm not sure how `currently_run` is used in the code (@rwaldron might be able to say more), but purely from a semantic standpoint, it seems like we should set the property to `false` for `firefox-57.0-linux`.

Similarly, this configuration of the `initially_loaded` property may cause results to be displayed for both versions of Firefox. It's less clear to me if that's desirable--on the one hand, we'd like to show the history data, but on the other hand, we probably don't need two stable releases of the same browser on the front page.

Reply:
> it seems like we should set the property to false for firefox-57.0-linux.

Unless we plan to run them both, then yes, it should be. 

I forgot that this file is coupled tightly with different parts of the system. Thanks for taking a closer look at this.

Reply:
Good call. I've updated the file accordingly. Let's see if this gives us a successful run before committing this.